integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/HttpEndpointTestCase.java

```diff
@@ -32,53 +32,46 @@
 import org.wso2.esb.integration.common.clients.endpoint.EndPointAdminClient;
 import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
 import org.wso2.esb.integration.common.utils.ESBTestConstant;
-import org.wso2.esb.integration.common.utils.servers.axis2.SampleAxis2Server;
 
-import javax.xml.stream.XMLStreamException;
 import java.io.File;
 import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.io.StringReader;
 import java.io.StringWriter;
-import java.net.MalformedURLException;
 import java.net.URL;
 import java.rmi.RemoteException;
 import java.util.Arrays;
 import java.util.List;
+import javax.xml.stream.XMLStreamException;
 
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertTrue;
 import static org.testng.Assert.fail;
 
-public class HttpEndpointTestCase extends ESBIntegrationTest {
 
-    private SampleAxis2Server axis2Server1 = null;
+public class HttpEndpointTestCase extends ESBIntegrationTest {
 
-    private final String ENDPOINT_NAME = "HTTPEndpointTest";
+    private final String ENDPOINT_NAME = "HTTPEndpointTestEP";
     private EndPointAdminClient endPointAdminClient;
-
-    private static final String studentName = "automationStudent";
-    private static final String updateStudentName = "automationStudent2";
+    private  static final String CUSTOMER_API_CONTEXT = "customerService";
+    private static final String RESOURCE_CONTEXT = "/customer";
+    private static final String customerId = "8fa3fc1b-f63c-4b21-8aff-3ac684c74d97";
+    private static final String customerName = "John";
+    private static final String updateCustomerName = "Emma";
+    private static final String getCustomerResponse = "<getCustomerResponse xmlns=\"http://ws.apache.org/ns/synapse\"><id>8fa3fc1b-f63c-4b21-8aff-3ac684c74d97</id><name>John</name></getCustomerResponse>";
 
     @BeforeClass(alwaysRun = true)
     public void init() throws Exception {
-
-            axis2Server1 = new SampleAxis2Server("test_axis2_server_9009.xml");
-            axis2Server1.start();
-            axis2Server1.deployService(ESBTestConstant.STUDENT_REST_SERVICE);
-
-
         super.init();
-        loadESBConfigurationFromClasspath(File.separator + "artifacts" + File.separator + "ESB" + File.separator + "endpoint" + File.separator + "httpEndpointConfig" + File.separator + "synapse.xml");
+        loadESBConfigurationFromClasspath(
+                File.separator + "artifacts" + File.separator + "ESB" + File.separator + "endpoint" + File.separator
+                        + "httpEndpointConfig" + File.separator + "synapse.xml");
```

Comment:
Can we please move configuration loading part out? Now we load all configs at start, and then run all the tests.